tools/project_manager/show_word_context.php

```diff
@@ -81,6 +81,22 @@
     echo "</a>";
     echo "</p>";
 
+    echo "<form method='GET'>";
+    echo "<input type='hidden' name='projectid' value='$projectid' />";
+    echo "<input type='hidden' name='word' value='$encWord' />";
+    echo "<input type='hidden' name='layout' value='$layout' />";
+    echo "<input type='hidden' name='frame' value='left' />";
+    echo "<label for='wordInstancesSelect'>" . _("Number of word context results: ") . "</label>";
+    echo "<select id='wordInstancesSelect' name='wordInstances' style='margin-left: 2px;'>";
+    foreach(range(10, 100, 10) as $option) {
+        echo "<option value='$option'" . ($option == $wordInstances ? "selected" : "") . ">$option</option>";
+    }
+    echo "</select>";
+    echo "<br />";
+    echo "<input type='submit' value='" . _("Update") . "' />";
```

Comment:
I'd either leave the "Update" button on the same line as the pulldown, or make it an auto-update pulldown. When I tested it, my eyes didn't immediately see the update button, because it was all the way back at the beginning of the line, hidden under existing text. (Not sure whether the auto-update is more appropriate, or the "execute" button, from a UX design standpoint.)

Reply:
Sure, I think updating automatically is better.